docs/helper_gui.rst

```diff
@@ -0,0 +1,152 @@
+helper_gui
+==========
+Regolith helpers can add to, list from, or update existing collections in your databases. While helpers can be accessed through command line/terminal, the Helper gui is a more comfortable and accessible way to use them.
+
+As with regular regolith, `helper_gui`  must be run from a directory containing a regolithrc.json file and it will use the config parameters it finds in there.  To use the gui, navigate to such a directory and type `regolith_helper`, e.g.,
```

Comment:
you have to type `helper_gui` not `regolith_helper`.    I guess this suggests that you didn't get anyone to test out these insructions.   Please have someone follow the instructions and see how easy it is for them to follow them, then make modifications when you discover that something is not clear.